dev-aws/kafka-shared-msk/contact-channels/account-identity.tf

```diff
@@ -0,0 +1,8 @@
+# Consume from account-identity.legacy.account.events.compacted
+module "legacy_account_events_consumer" {
```

Comment:
We usually put the consumers near the source topic, so in a&i module, so that they keep track on who is consuming their data

Reply:
Done